strawberry/field.py

```diff
@@ -66,6 +66,7 @@ def __init__(
         permission_classes: List[Type[BasePermission]] = (),  # type: ignore
         default: object = UNSET,
         default_factory: Union[Callable[[], Any], object] = UNSET,
+        metadata: Optional[Mapping[Any, Any]] = None,
```

Comment:
is this how metadata is defined in dataclasses' type hints?

Reply:
Yes: https://github.com/python/typeshed/blob/master/stdlib/dataclasses.pyi#L129

Reply:
But I did add `Optional` :thinking:

Reply:
optional is fine :)